### PR TITLE
Cargo Rework - MK *why did I ever lison to people that were just powergamers?*

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Most exports now have a fall off 
All most all exports that have fall off now fall off every 100

## Why It's Good For The Game

Takes the baby wheels off cargo now that they have, so many many options to sell stuff and help make cargo work with other departments

## Changelog
:cl:
add: Most organs now will have drop off - Even the black market has its limits!
add: Market drop off is now 3x closer, depression is reall
add: Miasma, ye that one, halfed it selling too.
add: Halfs most gasses that sell if not by 100 fold - Noable
add: Nryla now sells for a bit more.
add: Nerfs Anitmatter selling
add: Most tools need a lot to be sold before market gets fed up with them
add: Almost all clothing will always have a market as selling off 300 pares of gloves is really normal. Lots of people just like gloves
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
selling gases was alright because nobody did it

File: code/modules/cargo/exports/large_objects.dm [hunk 1]
```diff
@@ -171,13 +171,13 @@
...
 
-	worth += gases[/datum/gas/bz]*4
+	worth += gases[/datum/gas/bz]*2
 	worth += gases[/datum/gas/stimulum]*25
-	worth += gases[/datum/gas/hypernoblium]*1000
-	worth += gases[/datum/gas/miasma]*4
-	worth += gases[/datum/gas/tritium]*7
-	worth += gases[/datum/gas/pluoxium]*6
-	worth += gases[/datum/gas/nitryl]*30
+	worth += gases[/datum/gas/hypernoblium]*50
```
Comment (Hatterhat) [inline code/modules/cargo/exports/large_objects.dm]: hypernob is hard to make because it has that funky fusion-tier temperature/energy requirement or whatever, dabbing on it like this feels unnecessary

File: code/modules/cargo/exports/large_objects.dm [hunk 2]
```diff
@@ -171,13 +171,13 @@
...
+	worth += gases[/datum/gas/miasma]*1
+	worth += gases[/datum/gas/tritium]*5
+	worth += gases[/datum/gas/pluoxium]*4
+	worth += gases[/datum/gas/nitryl]*35
```
Comment (Hatterhat) [inline code/modules/cargo/exports/large_objects.dm]: remind me how nitryl's made again and your thought process that warrants bumping up the price a bit when you're throwing everything else out the door?

**Review by Hatterhat — CHANGES_REQUESTED** [review]
nitpicking about gases

File: code/modules/cargo/exports/large_objects.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/exports/large_objects.dm]: It is needed for other fusions, and tends to be sold less. By increasing its price it its a levy on new methods of fission being made or tried as a bad burn isnt as costly.

File: code/modules/cargo/exports/large_objects.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/exports/large_objects.dm]: if you make this you make tons of it. Meaning if you use 2n:1t:1bz you get 2 tanks out of it for over around 100k if not 250k. If you know how to do it, or know the waltz then you can p-easy make them as an atmos tech for cargo to gain well over 4 million by the one hour mark.
@Putnam3145 - Can you confirm/Enlighten me if im wrong on this?

Comment (Trilbyspaceclone) [reply code/modules/cargo/exports/large_objects.dm]: Furthermore, if you see above with Stim, being worth less the Nitryl. Meaning bad mixes or misstakes when making Nitryl only will be more costly depending on the goal in mind.
Just do to poor rates of sale dosnt mean it should be buffed or not reblanaced well others in its department are. - See Miasma, Toxins and Heaters
Its best to nip this one well its still a few-have-known. 233 bomb that gave 20k points was a issue that never needed to happen, so the can of 230 moles of hyper selling dosnt need that same issue as well.

Comment (Hatterhat) [reply code/modules/cargo/exports/large_objects.dm]: okay well this "hypernob waltz" you're talking about is some real funky stuff that only a few people know (and that small group is probably a circlejerk of atmos mains who refuse to teach anyone about assmos reactions)
and nitryl (burns lungs, slight stimulant effect(?)) being worse selling than stimulum proper (toxic, but a better stimulant effect) seems like a poor choice
why would central go "haha i really like this lung burning gas that is only slightly more useful than this funky stimulum gas that only /slightly/ kills our troops"

**Comment by ghost** [comment]
thoughts: if you're going to nerf gas selling you're babying cargo to an unfortunate degree. the more you try to cock strats that require skill/interdepartmental interaction/knowledge/etc the more you're coddling cargo and making it braindead easy to make money. otherwise whatever. it's not my choice if you want to make things less rewarding for going out of your way to do something difficult


tl;dr: why are we making exotic and difficult moneymaking methods worse? should difficult moneymaking methods not be more rewarding than afk farm tier methods?

**Comment by Trilbyspaceclone** [comment]
@kappa-sama Most gas selling is low skill, mass made riches based off one persons findings its a waltz you do, after someone teaches you or you find out how to make your own dance.
You can get endless gas via miners thus theirs endless fussion, ect ect. Plux/trit/miasma are all AFK farms with the SM/Rad collections or a monkey farm.
Passive income in forms of gas is not what we want.

**Comment by necromanceranne** [comment]
Here is a nice gauge for cargo balance built right in.

If you can buy a bicycle within an hour, you should not be able to do that.

Pretty easy, no?

**Review by Ghommie — CHANGES_REQUESTED** [review]
Keep the atmos export changes in a separate PR.

**Review by Ghommie — APPROVED** [review]
